### PR TITLE
Add support for Python 3.5

### PR DESCRIPTION
**Pull request by maryokhin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: python
 python:
-  - "2.6"
-  - "2.7"
+  - 2.6
```
Comment (maurizi) [inline .travis.yml]: You'll need to update the `exclude` configuration block below to remove the quotes around `"2.6"` to get the Travis CI builds to pass.

**Comment by maurizi** [comment]
@maryokhin Thanks for the PR!

Python 3 support has been desired by a number of folks for a while now, so I think we'll be happy to merge this if the Travis CI builds pass.

**Comment by maryokhin** [comment]
Yep, no problem @maurizi. Quick question though, Python 3.5 is failing of Django 1.7 because of this [bug](http://www.thefourtheye.in/2015/02/python-35-and-django-17s-htmlparseerror.html). Should I upgrade the test matrix to test on Django 1.7.11 or just exclude the Django 1.7 + Python 3.5 row from the test matrix?

**Comment by maurizi** [comment]
Upgrading Django seems like the better option.

**Comment by jwalgran** [comment]
According to this chart, Django 1.7 is not supported under Python 3.5.

https://docs.djangoproject.com/en/1.7/faq/install/#what-python-version-can-i-use-with-django

We should exclude it from our matrix

```yaml
matrix:
  exclude:
    - python: 3.5
      env: DJANGO=1.7

**Comment by maryokhin** [comment]
A bit confused by all the string stuff in Python 2/3, but trying to figure it out. Can you mb explain what the sanitisation of strings and `_safe_utf8_stringify` does, @jwalgran?

**Comment by maurizi** [comment]
@maryokhin As noted on the Python website [`csv.DictWriter on Python 2 does not support unicode`](https://docs.python.org/2/library/csv.html).  So the purpose of `_safe_utf8_stringify` is to ensure that all unicode string content gets encoded as UTF-8 before being added to the CSV.

This may not be necessary on Python 3, as the docs do not have the same note about not supporting unicode that they do on the Python 2 version.

**Pull request merged**